CHANGELOG.md

```diff
@@ -2,464 +2,311 @@
+    storage restrictions.
 
-- UI
-  - Implement HTML5 Console for RHEVM and VMware
-  - Menu plugins for external sites
-  - Orchestration templates UI updates
-  - HAML and I18n strings 100% completed in views
-- Automate
-  - New retirement workflow
+#### User Interface
+- HTML5 Console for RHEVM, VMware, and OpenStack
+- Menu plugins for external sites
+- HAML and I18n strings 100% completed in views
+- Multi-character set language support
+- Can now set the locale for both server and user
+- Repository Editor using AngularJS
+- Charting updates: jqPlot, default charts, chart styling, donut chart support
+- Updates to form buttons with Patternfly
+- Tagging support and toolbars on list views
+- Login screen converted to Bootstrap / Patternfly
+- Header, navigation, and outer layouts converted to Bootstrap / Patternfly
+- Advanced search converted to Bootstrap / Patternfly
+- Schedule editor converted to AngularJS
+- UI Customizations with Less
+- Dashboard tabs updated
+- Replaced many legacy Prototype calls with jQuery equivalents.
+
+#### Rest API 
+- Enhancement to /api/providers to support new provider class
+- Providers CRUD
+- Refresh via /api/providers
+- Tag Collection /api/tags 
+- Tag Management (assign and unassign to/from resources)
+- Foundational
+  - Virtual attribute support  
+  - Id/Href separation
+- Policy Management: Query policy and policy profiles conditions
+- VM Management
+  - Custom Attributes
+  - Add LifeCycle Events
+  - Start, stop, suspend, delete.
+- Total parity with SOAP API
+- Accounts sub-collection /api/vms/#/accounts
+- Software sub-collection /api/vms/#/software
+- Support for external authentication (httpd) against an IPA server.  
+
+#### Automate 
+- Enhanced UI import to allow granularity down to the namespace.
+- Cloud Objects exposed to Automate. 
+- Allow Automate methods to override or extend parameters passed to provider by
+  updating the clone_options during provisioning.  
+- New service model for CloudResourceQuota.
+- Exposed relationships through EmsCloud and CloudTenant models.
+- Exposed cloud relationships in automate service models.
+- Persist state data through automate state machine retries.
+- Moved auto-placement into an Automate state-machine step for Cloud and Infrastructure provisioning.
+- Added common "Finished" step to all Automate state machine classes.
+- Added eligible_* and set_* methods for cloud resources to provision task service model.
+- Ability to specify zone for web service automation request
+- Ability to override request message
+- Updated provisioning/retirement entry point in a catalog item or bundle.
+- Disabled domains clearly marked in UI.
+- Automate entry point selection reduced to state machine classes.
+- Services
+  - Dialog seeding for imports
+  - Service provisioning request overrides
+- Retirement
+  - New workflow
   - Added edit/copy/remove for Orchestration Templates
   - Enabled CloudFormation Provisioning through Services
```

Comment:
"Added edit/copy..." and "Enable Cloudformation..." are already covered under the Orchestration section and definitely do not belong under Retirement.